include/bluetooth/audio/vocs.h

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_VOCS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_VOCS_H_
+
+/**
+ * @brief Volume Offset Control Service (VOCS)
+ *
+ * @defgroup bt_gatt_vocs Volume Offset Control Service (VOCS)
+ *
+ * @ingroup bluetooth
+ * @{
+ *
+ * The Volume Offset Control Service is a secondary service, and as such should not be used own its
```

Comment:
I actually don't have any auto-split in my editor, so any such changes I've done has been intentional. Personally I don't find liken of length 100 to be any less readable, so it is, as you say, subjective :) 

From: https://www.phoronix.com/scan.php?page=news_item&px=Linux-Kernel-Deprecates-80-Col

> Yes, staying withing 80 columns is certainly still _preferred_. But it's not the hard limit that the checkpatch warnings imply, and other concerns can most certainly dominate

So from the Linux Kernel point of view, 80 should still be preferred. Going forwards I'll try to keep it at max 80, unless to avoid ugly breaks :)